template/.pre-commit-config.yaml.j2

```diff
@@ -18,6 +18,7 @@ repos:
     rev: eeee35a89e69d5772bdee97db1a6a898467b686e # 1.0
     hooks:
       - id: fmt
+        entry: cargo +nightly fmt
```

Comment:
question: Is overriding the entry key supported in non-local hooks? Because https://github.com/doublify/pre-commit-rust/issues/15 seems to suggest it is not.

If this is the case, we should consider using a local hook instead.

Reply:
I personally have to admit I don't know, I don't use pre-commit :see_no_evil:

Reply:
+1 for replacing pre-commit-rust altogether